ci/tests.sh

```diff
@@ -2,18 +2,19 @@
 
 # Runs the test suite for the travis build
 #
-# If COVERALLS is true, then we report test coverage to coveralls.
+# If EXTRA_CI_CHECKS is true, then we report test coverage to coveralls, and run
+# the mix formatter to ensure all files are formatted.
 #
-# This script could be used for local testing as long as COVERALLS is not set.
+# This script could be used for local testing as long as EXTRA_CI_CHECKS is not set.
 
 export MIX_ENV=test
 
-if [ "$COVERALLS" = true  ]
+if [ "$EXTRA_CI_CHECKS" = true  ]
 then
+  echo "Checking that all code is formatted"
```

Comment:
Looking at Travis output, I think this would be a little easier to read in the log:

```
Checking code format (`mix format --check-formatted`) ...
```